docs/get-started/getting-started-alluxio.md

```diff
@@ -0,0 +1,146 @@
+---
+layout: page
+title: Alluxio
+nav_order: 5
+parent: Getting-Started
+---
+# Get Started with RAPIDS with Alluxio
+
+Alluxio is a data orchestration platform brings your data closer to compute across clusters,
+regions, clouds, and countries. This guide will go through how to set up the RAPIDS Accelerator
+for Apache Spark 3.0 with Alluxio and on premise cluster.
+
+## Prerequisites
+
+Assuming that user has successfully run the RAPIDS Accelerator with on premise cluster according
+to [this doc](getting-started-on-prem.md)
+
+This guide takes Yarn cluster as an example and assumes that there are 2 datanodes and 1 namenode in Yarn cluster, and their hostnames are respectively
```

Comment:
is there a reason we use namenode and datanode here vs nodemanager and resourcemanager?  the namenode and datanodes are part of the HDFS cluster, are we assuming yarn has that or is it not needed -> ie I thought our test was with swiftstack.

Reply:
sorry, My bad. they should be ResourceManager and NodeManager. Thx Tom